main.nf

```diff
@@ -109,7 +109,11 @@ params.precursor_mass_tolerance = 5
 params.fragment_bin_offset = 0
 params.fdr_threshold = 0.01
 params.fdr_level = 'peptide-level-fdrs'
+if (params.fdr_level=='psm-level-fdrs'){
```

Comment:
Could do that using another syntax:

```
fdr_level = (params.fdr_level == 'psm-level-fdrs') ? '' : '-'+params.fdr_level
```

it evaluates the expression to be true/false and then either takes the first (true case) or the second clause (false case). Its a bit more elegant to do it that way 👍